### DISCUSSION TITLE
GLTF export available from runtime

### DISCUSSION THREAD

**Discussion by weirdybeardyman** [godot-extended-libraries/godot-ideas#49] [Ideas]
I am working on a modelling and prototyping tool, and being able to export gltf at runtime would be invaluable. I think this would also be really useful for games that want modding support and community content creation support, for example something like townscaper where you can export the maps and 3d print them. 
Is it feasible to make the gltf export module accessible at runtime, not just from the editor?

**Comment by Xrayez** [comment]
I think this is possible, and @fire is likely working on GLTF export to work in exported projects, see godotengine/godot-proposals#3273.

Also, consider supporting godotengine/godot-proposals#1632 in general. 🙂

**Reply by fire** [reply]
I need to check but I think gltf2 export already is in 3.4 and bug fixes for CSG and etc are in 3.5.

**Reply by weirdybeardyman** [reply]
That's brilliant, thanks for the great work! How can I support those proposals?
